records_mover/db/postgres/sqlalchemy_postgres_copy.py

```diff
@@ -0,0 +1,153 @@
+# This file is vendored from
+# https://raw.githubusercontent.com/jmcarp/sqlalchemy-postgres-copy/01ef522e8e46a6961e227069d465b0cb93e42383/postgres_copy/__init__.py
+#
+# See https://github.com/jmcarp/sqlalchemy-postgres-copy/pull/14 to
+# address the issue on why we had to vendor it instead of creating a
+# transitive dpeendency.
+
+import six
+
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Mapper, class_mapper
+from sqlalchemy.sql.operators import ColumnOperators
+from sqlalchemy.dialects import postgresql
+
+__version__ = '0.5.0'
+
+def copy_to(source, dest, engine_or_conn, **flags):
```

Comment:
E302 expected 2 blank lines, found 1

Reply:
This file is vendored - unfortunately the quality gem seems to be a little messed up with different drivers not always understanding the exclude config.  Boo.  I fixed some of the other low-hanging fruit to make up for these.